README.md

```diff
@@ -1,70 +1,144 @@
+version: '3.8'
+
+services:
+  arkitektonika:
+    container_name: Arkitektonika
+    image: pierreschwang/arkitektonika:dev
+    restart: unless-stopped
+    volumes:
+      - ./data:/app/data
+    environment:
+      - LOG_LEVEL=DEBUG   # if debug logs should be printed to the console 
+    networks:
+      web:
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.arkitektonika.entrypoints=web"
+      - "traefik.http.routers.arkitektonika.rule=Host(`arkitektonika.pschwang.eu`)"
+      - "traefik.http.middlewares.arkitektonika-https-redirect.redirectscheme.scheme=https"
+      - "traefik.http.routers.arkitektonika.middlewares=arkitektonika-https-redirect"
+      - "traefik.http.routers.arkitektonika-secure.entrypoints=websecure"
+      - "traefik.http.routers.arkitektonika-secure.rule=Host(`arkitektonika.pschwang.eu`)"
+      - "traefik.http.routers.arkitektonika-secure.tls=true"
+      - "traefik.http.routers.arkitektonika-secure.tls.certresolver=cloudflare"
+      - "traefik.http.routers.arkitektonika-secure.service=arkitektonika"
+      - "traefik.http.services.arkitektonika.loadbalancer.server.port=3000"
+      - "traefik.docker.network=web"
+
+networks:
+  web:
+    external: true
+````
+
+`/app/data` is mounted to the host at `/data` as that folder contains persistent data.
+
+## Prune data
+Execute the start command with the prune flag to execute the prune routine:
+``yarn start:prod --prune``
+
 ## Set up Expiration
-Create a cron job that runs at whatever frequency you desire. As an example, this will run the pruning script every 12 hours:
+
+Create a cron job that runs at whatever frequency you desire. As an example, this will run the pruning script every 12
+hours:
+
 ```
-0 */12 * * * cd /srv/arkitektonika && /usr/bin/yarn run app:prune
+0 */12 * * * cd /srv/arkitektonika && /usr/bin/yarn start:prod --prune
 ```
 
+Or with a docker-compose configuration:
+````
+0 */12 * * * cd /srv/arkitektonika && docker-compose run arkitektonika node app/launch.js --prune
+````
+
 ## Configuration
-* `public_url`: the public facing URL that this arkitektonika instance is available on, required by the pruning script; **default: http://localhost:3000**.
-* `multipart_param`: the name of the parameter that this arkitektonika instance accepts for file uploads; **default: schematic**.
-* `storage_folder`: the name of the folder where files will be stored; **default: schemata**.
-* `file_extension`: the file extension files uploaded will use; **default .schematic**.
-* `pruning.older_than_ms`: how long since in milliseconds since last access until a file will be eligible for expiration (if using the pruning script); **default 604800000** (7 days).
+
+````json
+{
+  // on which port should the application bind
+  "port": 3000,
+  // defines how old records must be to be deleted by the prune script (in ms)
+  "prune": 1800000,
+  // maximum amount of iterations to obtain a unique download and deletion token
+  "maxIterations": 20
```

Comment:
Shift the explanation below the code box, json doesn't allow comments like that.